config/webpack.common.js

```diff
@@ -167,6 +167,7 @@ module.exports = (env) => {
         DATA_SOURCE: 'remote',
         BRAND_TYPE: 'Konveyor',
         NODE_ENV: 'production',
+        NO_AUTH: 'false',
```

Comment:
You don't actually need `NO_AUTH` to be part of the `EnvironmentPlugin` if it's part of `FORKLIFT_ENV`. The other 3 variables are here because we want to be able to statically remove code based on those conditions (for example, webpack replaces `if (process.env['DATA_SOURCE'] === 'mock')` with `if (false)` and then deletes the unreachable code). They're also in `FORKLIFT_ENV` because that's a superset (having all the variables in the same place at run time is convenient).